build-feature.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 
 cd spring-graal-native-feature
-./mvnw clean package
```

Comment:
Using the Maven wrapper is a best practice that allows to build the project on environments without Maven installed, could you elaborate on the reasons why you need these changes?

Reply:
Well this downloads a file from the internet and writes to a particular location in the disk, in some environments that is no feasible

Reply:
OK I understand.